### ISSUE TITLE
cool project!

### ISSUE THREAD

**Issue by aquynh** [Worobow/MakeLib#1]
this looks like a very interesting tool! will you provide more docs & samples on how to use it?

keep it up, cheers.

**Comment by Worobow** [comment]
Yes, I'm going to do more documentation and more detailed examples. 
Thank you!
